examples/svm.py

```diff
@@ -110,3 +110,10 @@ def svm_from_cfg(cfg):
 inc_value = svm_from_cfg(incumbent)
 
 print("Optimized Value: %.2f" % (inc_value))
+
+
+# We can also validate our results (though this makes a lot more sense with instances)
```

Comment:
@mfeurer do you want to simply change the dataset in `svm_from_cfg() `,
or do you want to use training and test instances in the scenario?
The first should be trivial but a bit hacky;
the latter would be the intended way from SMAC's perspective, but harder to understand (/more complicated)